### PR TITLE
Updated Tabs documentation.

### PR DESCRIPTION
**Pull request by dialpad-drew**
## Description
This PR does the following:
* Updates Tabs documentation
* Namespaces the examples' `ids` to avoid instances in which one example's tabs could control another's tab-panels
* Removes background colors from inverted styles as they're unnecessary
* Moves accessibility documentation into `tabs.json`
* Wordsmiths some of the documentation to use common wording, phrasing, and voice as other components' documentation

**Note**: you may see a defect in which the paragraphs wrap "early." That's expected and will be fixed in another story.

screenshot-localhost_4000-2021 03 30-17_59_18

### CODE REVIEW

**Comment by dialpad-drew** [comment]
@braddialpad on top of just a general look at things, would you mind being my second set of eyes to make sure I didn't misinterpret or misstate any of your class or a11y documentation as I was editing it?

File: docs/_data/tabs.json [hunk 1]
```diff
@@ -2,27 +2,90 @@
...
     {
       "class": "d-tab",
-      "desc": "Default tab style"
+      "applies": "N/A",
+      "description": "Styles each tab."
     },
     {
       "class": "d-tab--selected",
-      "desc": "Selected tab style"
+      "applies": "d-tablist",
```
Comment (ryandialpad) [inline docs/_data/tabs.json]: Should this apply to `d-tab`?

**Review by ryandialpad — APPROVED** [review]
w/ one comment

Great work!

File: docs/_data/tabs.json (hunk 1 above)
Comment (dialpad-drew) [reply docs/_data/tabs.json]: Totally. Good catch 🙌

**Pull request merged**